service/flannelconfig/v2/resource/flanneld/desired.go

```diff
@@ -303,7 +303,7 @@ func newDaemonSet(customObject v1alpha1.FlannelConfig, etcdCAFile, etcdCrtFile,
 							Name: "etcd-certs",
 							VolumeSource: corev1.VolumeSource{
 								HostPath: &corev1.HostPathVolumeSource{
-									Path: "/etc/giantswarm/g8s/ssl/etcd",
+									Path: "/etc/kubernetes/ssl/etcd",
```

Comment:
I can imagine this will not work out. What we rather should do is create a new version bundle and add the changes there. Then old clusters are fine and we can safely move on with newer ones. We need this smooth migration to get everything sorted eventually.

Reply:
Ok. I had a doubt about this and it sounds sensible. I'll withdraw this PR, learn the process to create new version bundle and come back with a new PR. Thanks!